master/buildbot/steps/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .cmake import CMake
```

Comment:
What is the reason why this is required? I can't think of any and this will make the steps accessible from multiple places.

Reply:
This reexport is to facilitate importing types from buildbot. This avoids lengthy imports like from buildbot.steps.cmake import CMake. 

I like :

```python
from zorg.buildbot.builders import ClangBuilder
from zorg.buildbot.builders import FlangBuilder
from zorg.buildbot.builders import PollyBuilder
```

but not

```python
from buildbot.reporters.generators.build import BuildStartEndStatusGenerator
from buildbot.reporters.generators.buildrequest import BuildRequestGenerator
from buildbot.reporters.generators.buildset import BuildSetStatusGenerator
from buildbot.reporters.generators.worker import WorkerMissingGenerator
from buildbot.reporters.message import MessageFormatterRenderable
```

The first way seems more intuitive and easier to use.

Reply:
(sorry for late reply - took some time to double check my comment)

Reply:
Unless buildbot.plugins has type information, I'm afraid no one will like to import plugins from here. If you don't like reexport, then we may need to find a way to add type information to buildbot.plugins

Reply:
Thanks for pointing this out. Indeed, buildbot.plugins will need improvement in this regard.